### PR TITLE
Create letterboxd-for-games.yml

### PR DESCRIPTION
**Pull request by danielgrijalva**
For [letterboxd-for-games](https://github.com/danielgrijalva/letterboxd-for-games) 🎮✨

### CODE REVIEW

File: _data/projects/letterboxd-for-games.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- react
+- reactjs
+- javascript
+- python
+- django
+- django-rest-framework
+- video games
+- gaming
+upforgrabs:
+  name: up for grabs :zap:
```
Comment (ritwik12) [inline _data/projects/letterboxd-for-games.yml:15]: @danielgrijalva Your build is failing and I guess it is due to ":" used in this line, can you please check by removing it.

Comment (ritwik12) [reply _data/projects/letterboxd-for-games.yml:15]: If this is the issue and you want to use this label name only then later we can ignore the build error.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/letterboxd-for-games.yml (hunk 1 above)
Comment (danielgrijalva) [reply _data/projects/letterboxd-for-games.yml:15]: hi @ritwik12, that's correct, I want to use that label name 😅

Comment (ritwik12) [reply _data/projects/letterboxd-for-games.yml:15]: Let me check for other errors.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@danielgrijalva I have merged this, please check if your project is coming on the site as it should be and let me know.

**Comment by ritwik12** [comment]
@danielgrijalva Hey, it isn't coming, The thing is it is treating : for dictionary values. What we can do is to revert it back and you can change the name of tag without ":". and give link https://github.com/danielgrijalva/letterboxd-for-games/labels/up%20for%20grabs%20%3Azap%3A

**Comment by ritwik12** [comment]
@danielgrijalva I have reverted it, please make the changes and send again. You can use the name "up for grabs zap" only if you want. Using ":" will fail the build.

**Comment by danielgrijalva** [comment]
@ritwik12 will do. I'll remove the emojis!

**Comment by ritwik12** [comment]
you can keep the emojis, link isn't effected by this. just change the name.

**Comment by ritwik12** [comment]
@danielgrijalva  Your link stays same, just change name of label in this file not in your repo. That way you can keep your emoji

**Comment by ritwik12** [comment]
https://github.com/danielgrijalva/letterboxd-for-games/labels/up%20for%20grabs%20%3Azap%3A

Use this link this time

**Comment by ritwik12** [comment]
I hope it clears things

**Comment by danielgrijalva** [comment]
oh ok, thanks I will do that instead

**Comment by danielgrijalva** [comment]
@ritwik12 I submitted a new PR. Can you check it? 😃

**Pull request merged**